lib/src/models/subreddit.dart

```diff
@@ -254,6 +255,15 @@ class SubredditRef extends RedditBase
     return null; // Shut the analyzer up.
   }
 
+  /// Returns a [Future<List<Moderator>>] of moderators.
+  Future<List<Moderator>> moderators() async {
+    return (await reddit.get(
```

Comment:
I'm pretty sure you're going to need to update the [Objector](https://github.com/draw-dev/DRAW/blob/master/lib/src/objector.dart) to parse the `Moderator` objects.